src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (const row of table.rows) {
+  row.cells[4].before(row.cells[1].cloneNode(true));
+}
```

Comment:
add empty line https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file